### PR TITLE
added homepage

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Chnage Logs

## Live preview link: 
https://deploy-preview-2--dreamy-thompson-b6907b.netlify.app/

### CODE REVIEW

File: styles/nav.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    display: flex;
+    padding: 0.6em;
+    justify-content: center;
+    list-style: none;
+    border-top: 1px solid var(--border-color);
+}
+
+.link{
+    padding: 0 0.6em;
+    margin: 0 0.3em;
```
Comment (HarshnaNHaswani) [inline styles/nav.css:42]: use rem here

File: styles/index.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    padding: 1.5em 0;
+}
+
+.product-category{
+    color: var(--primary-font-color);
+    opacity: 0.8;
+    padding: 0 1em;
+    font-size: 2em;
+    font-weight: 500;
+    margin-bottom: 1em;
```
Comment (HarshnaNHaswani) [inline styles/index.css:55]: use rem for margin/padding

**Review by HarshnaNHaswani — COMMENTED** [review]
Looks good to me! Nice work

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: styles/nav.css (hunk 1 above)
Comment (bishnoimukesh) [reply styles/nav.css:42]: I will take care of it.

**Pull request merged**